rasa/core/evaluation/stats.py

```diff
@@ -0,0 +1,111 @@
+import os
+from typing import Dict, Text, Union, Any, List, Tuple, TypedDict
+from pathlib import Path
+import json
+import numpy as np
+
+
+class Stats(TypedDict):
+    """A TypedDict for stats."""
```

Comment:
I'd maybe say "Represents statistics generated over success markers" or something that makes its purpose clearer, since I don't think we're intending for this to be generic